<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CShopdlg</class>
 <widget class="QWidget" name="CShopdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>money</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="color">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="cross">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="numOfMoney">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>money:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="numOfCross">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="numOfBoom">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyCross">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyColor">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="numOfColor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="boom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buyBoom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SHOP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
